### PR TITLE
Allow multiple IMG tags to be formatted

### PR DESCRIPTION
**Pull request by rdcx**
Allow multiple images to be formatted as IMG tags from one input array, this function calls HtmlHelper::image

Its basically to make things easier if you were using for example, glob() where an array could be passed straight in. Or if you did want for format options on each of them, then that also would be fine..

### CODE REVIEW

**Comment by dereuromark** [comment]
Not sure if this is core Material. This could easily reside in a plugin.

File: tests/TestCase/View/Helper/HtmlHelperTest.php [hunk 1]
```diff
@@ -337,6 +337,52 @@ public function testImageTag()
...
 
     /**
+     * testImages method
+     *
+     * @return void
+     */
+    public function testImages()
+    {
+        Router::connect('/:controller', ['action' => 'index']);
+        Router::connect('/:controller/:action/*');
```
Comment (markstory) [inline tests/TestCase/View/Helper/HtmlHelperTest.php]: Not sute you need routes to make image links.

File: src/View/Helper/HtmlHelper.php [hunk 2]
```diff
@@ -838,6 +838,61 @@ public function image($path, array $options = [])
...
+     *   `options` Reference HtmlHelper::image - options
+     *
+     * @param array $multipleImages Array of image paths or Array of path and options. See above for Multi Image structure.
+     * @return string of multiple image tags
+     */
+    public function images(array $multipleImages)
+    {
+        $images = '';
+
+        if (!empty($multipleImages)) {
```
Comment (dereuromark) [inline src/View/Helper/HtmlHelper.php]: This if is not necessary.

**Comment by markstory** [comment]
Historically, we haven't added methods like this as we try to focus on methods that can be used in the majority of applications our users build. I am not sure what this method is solving that a `foreach` couldn't. Do you frequently output glob lists of images in the applications you build?

**Comment by ross-chater** [comment]
Yeah that's a very true point, I noticed I've been using it a few times. Just wanted to see what you guys think and what your view was on how often it would be used.

Thanks

**Comment by ross-chater** [comment]
If there were more options on the images() function itself regarding group styling and div formation is this something that could be seen as more useful and used...

**Comment by jhli** [comment]
I encourage you to maje it a plugin and supply it to awesome cakephp list.

**Comment by ADmad** [comment]
I too don't think this belongs in core. It should be done as a community plugin.

**Pull request closed without merging**